app/templates/agenda.html

```diff
@@ -44,7 +44,11 @@ <h1>{{ start_date.strftime("%d/%m/%Y") }} - {{end_date.strftime("%d/%m/%Y") }}</
     {% for events_date, events_list in events.items() %}
     <div class="shown_dates">{{ events_date.strftime("%d/%m/%Y") }}</div>
         {% for event in events_list %}
-        <div class="event_line"> {{ event[0].start.time().strftime("%H:%M") }} - <a href="/event/view/{{ event[0].id }}">{{ event[0].title }}</a> , duration: {{ event[1] }}</div>
+        <div class="event_line">
+            {% set availability = 'Busy' if event[0].availability == True else 'Free' %}
+            <div class="{{ availability | lower }}" title="{{ availability }}"></div>
+            <div>{{ event[0].start.time().strftime("%H:%M") }} - <a href="/event/view/{{ event[0].id }}">{{ event[0].title }}</a> , duration: {{ event[1] }}</div>
```

Comment:
Escape the `event[0].title` (and everything else coming from the user) with `| safe`